### PR TITLE
1.7 changelog date and version bump

### PR DESCRIPTION
**Pull request by reaperhulk**
Prepping for the Monday morning release :)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,10 +1,8 @@
 Changelog
 =========
 
-1.7 - `master`_
-~~~~~~~~~~~~~~~
-
-.. note:: This version is not yet released and is under active development.
+1.7 - 2016-12-11
```
Comment (alex) [inline CHANGELOG.rst]: This isn't going to be released on the 11th.

Comment (reaperhulk) [inline CHANGELOG.rst]: Sigh, I even thought of this and looked at a calendar and managed to screw it up still.

**Comment by reaperhulk** [comment]
jenkins, retest this please

**Comment by reaperhulk** [comment]
jenkins, retest this please

**Comment by reaperhulk** [comment]
jenkins, retest this please

**Pull request merged**